src/core/lib/security/security_connector/tls/tls_security_connector.cc

```diff
@@ -406,22 +407,22 @@ grpc_error* TlsChannelSecurityConnector::ProcessServerAuthorizationCheckResult(
     gpr_asprintf(&msg,
                  "Server authorization check is cancelled by the caller with "
                  "error: %s",
-                 arg->error_details);
+                 arg->error_details->err_details().c_str());
```

Comment:
if you had another method in `grpc_tls_err_details` to return C string you wouldn't need to call `err_details().c_str()`

Reply:
fixed in the new PR.